docs/blitz/installation/installation.rst

```diff
@@ -0,0 +1,30 @@
+Installation
```

Comment:
lets remove this page as it is part of genie, which already has an installation page

Reply:
Health check and clean both has it though. Isnt it best if we keep it for sake of consistency?

Reply:
i've asked to remove it for both of these too

Reply:
Removed for blitz/clean/health alltogether